patches/server/0034-IntelliJ-Remove-Streams.patch

```diff
@@ -0,0 +1,603 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Bud Gidiere <sgidiere@gmail.com>
+Date: Mon, 10 Aug 2020 14:11:47 -0500
+Subject: [PATCH] IntelliJ Remove Streams
+
+Do not update, just regen.
+
+diff --git a/src/main/java/net/minecraft/server/BehaviorGate.java b/src/main/java/net/minecraft/server/BehaviorGate.java
+index 46e910581210421c8699637431804dc2f43eb4a6..945c7d0957e5fcdd4dc3b3ac95e274bdee23bf3f 100644
+--- a/src/main/java/net/minecraft/server/BehaviorGate.java
++++ b/src/main/java/net/minecraft/server/BehaviorGate.java
+@@ -19,9 +19,9 @@ public class BehaviorGate<E extends EntityLiving> extends Behavior<E> {
+         this.b = set;
+         this.c = behaviorgate_order;
+         this.d = behaviorgate_execution;
+-        list.forEach((pair) -> {
+-            this.e.a(pair.getFirst(), (Integer) pair.getSecond());
+-        });
++        for (Pair<Behavior<? super E>, Integer> pair : list) {
++            this.e.a(pair.getFirst(), pair.getSecond());
++        }
+     }
+ 
+     @Override
+@@ -62,7 +62,10 @@ public class BehaviorGate<E extends EntityLiving> extends Behavior<E> {
+         });
+         BehaviorController behaviorcontroller = e0.getBehaviorController();
+ 
+-        this.b.forEach(behaviorcontroller::removeMemory); // Paper - decomp fix
++        // Paper - decomp fix
++        for (MemoryModuleType<?> memoryModuleType : this.b) {
++            behaviorcontroller.removeMemory(memoryModuleType);
++        }
+     }
+ 
+     @Override
+diff --git a/src/main/java/net/minecraft/server/ChunkConverter.java b/src/main/java/net/minecraft/server/ChunkConverter.java
+index 5366314e5f889b5b8d7740bbd0f024d9b7b9d643..a238b269eec03e2d81688907329a04bbb07943cf 100644
+--- a/src/main/java/net/minecraft/server/ChunkConverter.java
++++ b/src/main/java/net/minecraft/server/ChunkConverter.java
+@@ -70,9 +70,9 @@ public class ChunkConverter {
+ 
+         World world = chunk.getWorld();
+ 
+-        ChunkConverter.g.forEach((chunkconverter_a) -> {
```

Comment:
Not a stream.

https://hg.openjdk.java.net/jdk8/jdk8/jdk/file/tip/src/share/classes/java/lang/Iterable.java#l72